cla_public/static-src/javascripts/modules/form-errors.js

```diff
@@ -262,11 +262,17 @@
       _.each(errorFields, addErrors);
 
       if(this.$form.data('error-banner') !== false) {
-        this.$form.closest('main').prepend(this.mainFormError({ errors: this.createErrorSummary(unattachedErrors)}));
-        if (GOVUK.getCookie("locale") == "cy_GB") {
-          $("title").prepend("Gwall: ");
+        if ($("#submit-button").length && $(".s-hidden input").val()) {
+          // LGA-1162 checking for hidden fields that are throwing errors and clearing them
```

Comment:
Idk if this is petty but could we remove the LGA-1162 from the comments? I was thinking the comments are pretty sufficient as a summary